src/bodypart.cpp

```diff
@@ -437,6 +444,52 @@ float bodypart::get_wetness_percentage() const
     return static_cast<float>( wetness ) / id->drench_max;
 }
 
+float bodypart::wound_adjusted_limb_value( const float val ) const
+{
+    double percent = static_cast<double>( get_hp_cur() ) /
+                     static_cast<double>( get_hp_max() );
+    if( percent > 0.75 ) {
+        return val;
+    }
+    percent /= 0.75;
+    return val * static_cast<float>( percent );
+}
+
+float bodypart::encumb_adjusted_limb_value( const float val ) const
+{
+    return val * 19.0f / ( 19.0f + get_encumbrance_data().encumbrance / 3.0f );
+}
+
+float bodypart::get_manipulator_score() const
+{
+    return id->manipulator_score;
+}
+
+float bodypart::get_manipulator_max() const
+{
+    return id->manipulator_max;
+}
+
+float bodypart::get_encumb_adjusted_manipulator_score() const
+{
+    return encumb_adjusted_limb_value( get_wound_adjusted_manipulator_score() );
+}
+
+float bodypart::get_wound_adjusted_manipulator_score() const
+{
+    return wound_adjusted_limb_value( get_manipulator_score() );
+}
+
+float bodypart::get_blocking_score() const
+{
+    return encumb_adjusted_limb_value( wound_adjusted_limb_value( id->blocking_score ) );
+}
+
+float bodypart::get_lifting_score() const
+{
+    return wound_adjusted_limb_value( id->lifting_score );
```

Comment:
i don't think so. maaaybe affected by parent bodypart, but i don't think it has the kind of access necessary in this scope.

Reply:
Pity. OK.

Reply:
Another query: Should this be affected by `STRONGBACK` or `BADBACK` as per lift strength (`get_lift_str()`)? (If so, then eventually it may be done through those traits incorporating a modifier to lifting score.)

Reply:
perhaps in the Character:: scope